crypto/arm64cpuid.pl

```diff
@@ -104,6 +104,22 @@
 	ret
 .size	_armv8_eor3_probe,.-_armv8_eor3_probe
 
+.globl	_armv8_sve_probe
+.type	_armv8_sve_probe,%function
+_armv8_sve_probe:
+	AARCH64_VALID_CALL_TARGET
+	.long	0x04a03000	// eor z0.d,z0.d,z0.d
```

Comment:
Judging by #18086 this, consider using .inst directive?

Reply:
it looks like there are many similar instances to fix in the same file if we need to switch to .inst altogether. 

but i will change the sve probe in this case.